guides/common/modules/proc_enabling-and-disabling-repositories-on-hosts.adoc

```diff
@@ -0,0 +1,11 @@
+[id="Enabling_and_Disabling_Repositories_on_Hosts_{context}"]
+= Enabling and Disabling Repositories on Hosts
+
+Use this procedure to enable and disable repositories on hosts for {Project}.
+
+.Procedure
+. In the {Project} web UI, navigate to *Hosts* > *All Hosts*,
+. Select the host name.
+. Click the *Content* tab.
+. Click the *Repository Sets* tab.
+. Click the vertical ellipsis to choose *Override to disabled* or *Override to enabled* to disable or enable repositories on hosts.
```

Comment:
Also, maybe it could be specified where the vertical ellipsis is located?
```suggestion
. Click the vertical ellipsis.
. Choose *Override to disabled* or *Override to enabled* to disable or enable repositories on hosts.
```